e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -73,7 +76,7 @@ test.describe('Notebook section tests', () => {
 
         // Create Notebook
         await createDomainObjectWithDefaults(page, {
-            type: 'Notebook',
+            type: NOTEBOOK_NAME,
             name: "Test Notebook"
```

Comment:
Let's remove the name attribute and get the name from the returned object i.e:

```js
notebookObject = createDomainObjectWithDefaults(page, {
    type: NOTEBOOK_NAME
});
```

And then we can use `notebookObject.name` anywhere we need to reference it. This makes the notebook name unique in the case of a persistent environment e.g.: running against couchdb. You can declare `let notebookObject;` above the `beforeEach` to make it available to all the tests in the suite.

Reply:
Ok, I removed name, but we're not using the notebookObject anywhere yet in a couple of these describe blocks, so I'm not assigning it to anything yet, we can do that in the future when we add more tests and actually need the object.